src/components/OptionsDialog/index.js

```diff
@@ -0,0 +1,111 @@
+import $ from 'jquery'
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import ReactDOM from 'react-dom'
+
+import FieldHelp from '../FieldHelp/index'
+
+import {
+  getSSHKey,
+  saveSSHKey,
+} from './actions'
+
+import { msg } from '../../intl'
+
+class OptionsDialog extends React.Component {
+
+  constructor (props) {
+    super(props)
+    this.state = {
+      sshKey: props.optionsDialog.get('sshKey') || '',
+    }
+    this.onSSHKeyChange = this.onSSHKeyChange.bind(this)
+    this.onSaveClick = this.onSaveClick.bind(this)
+  }
+
+  componentDidMount () {
+    const dom = ReactDOM.findDOMNode(this)
+    $(dom).on('shown.bs.modal', function (e) {
+      if (this.props.userId) {
+        this.props.onOpen()
+      }
+    }.bind(this))
+  }
+
+  componentWillReceiveProps (nextProps) {
+    this.setState({ 'sshKey': nextProps.optionsDialog.get('sshKey') })
+  }
+
+  onSSHKeyChange (event) {
+    this.setState({ sshKey: event.target.value })
+  }
+
+  onSaveClick () {
+    this.props.onSave({ key: this.state.sshKey, sshId: this.props.optionsDialog.get('sshId') })
+  }
+
+  render () {
+    let { oVirtApiVersion } = this.props
```

Comment:
`const`. Please prefer `const` over `let` where (reasonably) possible. It improves readability greatly since reader is not forced to scan following part of scope for other assignments.